pluginbase.py

```diff
@@ -1,37 +1,28 @@
+import typing
+
 from abc import ABC
 from abc import abstractmethod
 from typing import Optional
 
-import perf
-
-from task import Task
-from testConfig import TestConfig
 from tftbase import PluginOutput
 from tftbase import PluginResult
 from tftbase import TestMetadata
 
 
-class PluginTask(Task):
```

Comment:
Why do we need to move this to Task? And my understanding is that this causes issues.

Reply:
The issue is the order of dependencies of certain modules, which this commit fixes.

It could also be fixed while leaving  `PluginTask` inside `pluginbase` module. But the issue is there regardless, and indeed, moving it doesn't cause nor fix the issue.

This commit moves the `PluginTask`, because it seems to make sense to have trivial types in `tftbase`. The triviality of those types, makes it a non-issue to have them at the very basic module (`tftbase`) and having them accessible to most other modules. Also, while `PluginTask` was previously beside the `Plugin` (in `pluginbase`), it's now beside the `Task` (in `tftbase`). So it's still beside a thing, to which it is closely related.

Reply:
see https://github.com/wizhaoredhat/ocp-traffic-flow-tests/pull/43#discussion_r1635914486

(lets follow up there).